app/src/main/java/com/watermark/community_app/communityapp/ContentManager.java

```diff
@@ -165,7 +167,7 @@ private PostData processPostData(CDAEntry post, CDAArray assets) {
             String imageUrl = (String) map.get("url");
             URL url = null;
             try {
-                //TODO: Setting the background to an image is slow. Commenting out while developing
+                // TODO: Setting the background to an image is slow. This is causeing an out of memory exception.
```

Comment:
We need to use the image loader Library Glide. I can accept this PR and just pull in more libraries later.